functions/log/main.py

```diff
@@ -66,3 +72,14 @@ def get_log_entries(request):
 
     return 'Done!'
 # [END functions_log_retrieve]
+
+
+# [START functions_log_stackdriver]
+def process_log_entry(data, context):
+    data_buffer = base64.b64decode(data['data'])
+    log_entry = json.loads(data_buffer)['protoPayload']
+
+    print(f"Method: {log_entry['methodName']}")
```

Comment:
Since this is Python 3 only we're clear to use f strings, but does our lint support it now? It was giving trouble with these before, right?